torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -526,8 +548,13 @@ void ONNXAssignOutputShape(
   }
 }
 
-void ONNXShapeTypeInference(std::shared_ptr<Graph>& graph, int opset_version) {
-  ONNXShapeTypeInference(graph->block(), opset_version);
+void ONNXShapeTypeInference(
+    std::shared_ptr<Graph>& graph,
+    const ParamMap& params_dict,
+    int opset_version) {
+  for (auto n : graph->nodes()) {
```

Comment:
Why changing block inference to node inference?

Reply:
thanks for catching, looks like out dated merge, reverting to calling block version.